Packs/PaloAltoNetworks_IoT/Scripts/iot_alert_post_processing/iot_alert_post_processing.py

```diff
@@ -0,0 +1,35 @@
+import demistomock as demisto
+
+
+def _get_incident():
+    return demisto.incidents()[0]
+
+
+def iot_resolve_alert():
+    incident = _get_incident()
+
+    _id = ""
+    for label in incident['labels']:
+        if label['type'] == 'id':
+            _id = label['value']
+            break
+
+    args = demisto.args()
+    close_reason = args.get('closeReason')
+
+    demisto.executeCommand('iot-resolve-alert', {
+        'id': _id,
+        'reason': f'resolved by XSOAR incident {incident["id"]}',
+        'reason_type': 'Issue Mitigated' if close_reason == "Resolved" else 'No Action Needed'
+    })
+
+
+def main():
+    try:
+        iot_resolve_alert()
+    except Exception as ex:
+        demisto.error(f'Failed to execute iot-alert-post-processing. Error: {str(ex)}')
```

Comment:
First of all, this is a Post Processing. I tried return_error or some sort in the past, and I ended up wasting few hours to debug because
1. it was NOT printing out anything
2. It was NOT failing the post processing because the xsoar incident could still be closed

demisto.error in post-processing is the only reliable thing that can let me at least debug the issue.